docs/admin/repo/add.mdx

```diff
@@ -17,12 +17,42 @@
 
 ## Troubleshooting
 
+### Repositories not showing up
+
 If your repositories are not showing up, check the site admin **Repositories** page on Sourcegraph (and ensure you're logged in as an admin).
 If nothing informative is visible there, check for error messages related to communication with your code host's API in the logs from:
 
 - [Docker Compose](/admin/deploy/docker-compose/) and [Kubernetes](/admin/deploy/kubernetes/): the logs from the `repo-updater` container/pod
 - [Single-container](/admin/deploy/docker-single-container/): the `sourcegraph/server` Docker container
 
+### Repository not cloning or updating
+
 If your repositories are showing up but are not cloning or updating from the original Git repository:
 
 - Follow the instructions for [troubleshooting a repository that is not being updated](/admin/how-to/repo-not-updated)
+
+### Sourcegraph can't connect to your code host
+
+If Sourcegraph displays errors when syncing repositories from your code host, please click the "Test connection" button on the code host configuration page.
+
+If the connection test fails with errors like:
+
+> Connection check failed: request failed: context deadline exceeded
+>
+> This error might indicate that the code host is not reachable from your Sourcegraph instance.
```

Comment:
with your other PR, this message will change slightly right? 

Do we want to reflect that here?

Reply:
No, this note takes into account the updating wording in https://app.graphite.dev/github/pr/sourcegraph/sourcegraph/62977/cmd-frontend-add-note-that-clarifies-that-your-code-host-needs-to-be-accessible-over-the-network-to-test-connection-button. (The "this error might indicate ..." message never existed before.)